src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -90,6 +88,8 @@ sortObjects(ViewType &view)
       bin_sort(view, CompType(n / 2, result.min_val, result.max_val), true);
   bin_sort.create_permute_vector();
   bin_sort.sort(view);
+  // FIXME Kokkos::BinSort is currently missing overloads that an execution
+  // space as argument
```

Comment:
Need to report that on the Kokkos side and fix

Reply:
Please link it here when you open the issue there.